.circleci/config.yml

```diff
@@ -5,9 +5,16 @@ references:
     run:
       name: Docker login, build, and push
       command: |
-        docker login quay.io -u="reactiveops+circleci" -p="${quay_token}"
         docker build -f Dockerfile -t quay.io/reactiveops/rbac-manager:$DOCKER_BASE_TAG .
-        docker push quay.io/reactiveops/rbac-manager:$DOCKER_BASE_TAG
+
+        if [[ -z $CIRCLE_PR_NUMBER ]]; then
```

Comment:
i'm not sure CIRCLE_PR_NUMBER would be present in the release workflow. Perhaps it could be explicitly set in that case?

Reply:
I think this is WAI - if `CIRCLE_PR_NUMBER` is unset, we `docker push`. `CIRCLE_PR_NUMBER` will only be set when running `test` and `build` on forked PRs.

https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables